### ISSUE TITLE
Is this possible to use double font?

### ISSUE THREAD

**Issue by Yave-Yu** [Aegisub/Aegisub#382]
Set a latin font as main font, then set CJK font as fallback font.

**Comment by animefn** [comment]
No, fallback fonts are something your player falls back to when it falls to locate the font of the pre-defined styles in the .ass file.

Just include the fonts as attachments in your mkv and you will not need fallback fonts. Things will appear correctly with the right fonts.
